src/ngraph/serializer.cpp

```diff
@@ -1911,11 +1911,29 @@ shared_ptr<Node> JSONDeserializer::deserialize_node(json node_js)
         }
         case OP_TYPEID::TopK:
         {
-            auto top_k_axis = node_js.at("top_k_axis").get<size_t>();
-            auto k = node_js.at("k").get<size_t>();
-            auto compute_max = node_js.at("compute_max").get<bool>();
-            auto target_type = read_element_type(node_js.at("index_element_type"));
-            node = make_shared<op::TopK>(args[0], top_k_axis, target_type, k, compute_max);
+            if (op_version == 0)
+            {
+                auto top_k_axis = node_js.at("top_k_axis").get<size_t>();
+                auto k = node_js.at("k").get<size_t>();
+                auto compute_max = node_js.at("compute_max").get<bool>();
+                auto target_type = read_element_type(node_js.at("index_element_type"));
+                node = make_shared<op::TopK>(args[0], top_k_axis, target_type, k, compute_max);
+            }
+            else if (op_version == 1)
+            {
+                const auto axis = node_js.at("axis").get<size_t>();
+                const auto mode = node_js.at("mode").get<op::v1::TopK::Mode>();
```

Comment:
Because of a discussion that was started in another PR, I'm going to leave it as is for now, and changed it later based on the results of that conversation.